Client/Frontend/Home/PocketFooterView.swift

```diff
@@ -5,12 +5,13 @@
 import UIKit
 import Shared
 
-class PocketFooterView: UICollectionReusableView, ReusableCell, ThemeApplicable {
+class PocketFooterView: UICollectionReusableView, ReusableCell, FeatureFlaggable, ThemeApplicable {
     private struct UX {
         static let fontSize: CGFloat = 12
         static let mainContainerSpacing: CGFloat = 8
     }
 
+    private let wallpaperManager: WallpaperManagerInterface
```

Comment:
Well, annoyingly I need to revert this since `featureAvailable` isn't on the interface, although it's a public method. We'll leave that be for now I guess
```suggestion
    private let wallpaperManager: WallpaperManagerInterface
```